src/service/xhr-impl.js

```diff
@@ -139,6 +139,9 @@ export class Xhr {
             ` ${ALLOW_SOURCE_ORIGIN_HEADER} header`);
       }
       return response;
+    }).catch(reason => {
+      user.assert(false, 'Fetch failed %s: %s', input,
```

Comment:
sorry switched this out with `.then(succHandler, errHandler)` which i believe should be more similar to the original behavior. (just making sure you saw the latest change)